Sources/AblyChat/Messages.swift

```diff
@@ -0,0 +1,74 @@
+import Ably
+
+public protocol Messages: AnyObject, Sendable, EmitsDiscontinuities {
```

Comment:
Maybe lift `AnyObject` requirement? Because it's a pain to add vars to the implementation. It can't be `var` + `class` + `sendable` at the same time:

```
Stored property 'var' of 'Sendable'-conforming generic class 'Class' is mutable
```

Unanswered discussion about the problem - https://forums.swift.org/t/is-this-sendable-warning-correct/55632

Reply:
It's marked as `var … { get }`, which means you can implement it as a `let` in a class.

Reply:
What if a `private var` is needed?

Reply:
There's a general question to answer, which is "how are we going to implement a `Sendable` type that internally manages mutable state"? And I don't know what the answer to that is going to be, because I haven't done it before. It might be that we end using actors (and hence possibly having to make more of our interface `async`, or it might be that we implement some `@unchecked Sendable` types that manage the state with locks. But I don't think that removing the `AnyObject` requirement will really help us.